docs/book/src/developer/tilt.md

```diff
@@ -458,3 +460,13 @@ syntax highlighting and auto-formatting. To enable it for Tiltfile a file associ
   "Tiltfile": "starlark",
 },
 ```
+
+## Using Podman
+
+[Podman](https://podman.io) can be used instead of docker by following these actions:
+
+1. Enable the podman unix socket (eg. `systemctl --user enable --now podman.socket` on Fedora)
+1. Set `build_engine` to `podman` in `tilt-settings.yaml`
+1. Define the env variable `DOCKER_HOST` to the right socket while running tilt (eg. `DOCKER_HOST=unix:///run/user/$(id -u)/podman/podman.sock tilt up`)
+
+NB: The socket defined by `DOCKER_HOST` is used only for the `hack/tools/tilt-prepare` command, the image build is running the `podman build`/`podman push` commands.
```

Comment:
I think this is required to use CAPD as well, is that correct?

Reply:
I don't know much about CAPD but I think it would need a bit more work to support podman.

If I add `docker` to my `enable_providers`, the controller fails while starting containerd:
```
[event: pod capd-system/capd-controller-manager-8585cbbd48-s5cfk] Error: failed to create containerd task: failed to create shim: OCI runtime create failed: container_linux.go:380: starting container process caused: process_linux.go:545: container init caused: rootfs_linux.go:87: creating device nodes caused: no such file or directory: unknown
```

I know containerd can run inside podman because kind is working fine with podman, but I think it requires some adjustments to the configs/args.

Reply:
I'm marking this conversation to resolved as the podman support in CAPD is out of scope here